onnxruntime/test/python/onnxruntime_test_python_cudagraph.py

```diff
@@ -50,7 +94,8 @@ def testRunModelWithCudaGraph(self):
             io_binding.bind_ortvalue_input("X", x_ortvalue)
             io_binding.bind_ortvalue_output("Y", y_ortvalue)
 
-            # One regular run for the necessary memory allocation and cuda graph capturing
+            # Two regular runs for the necessary memory allocation and cuda graph capturing
+            session.run_with_iobinding(io_binding)
```

Comment:
I think we only expect the user to run once - we internally did all the multiple runs here - https://github.com/microsoft/onnxruntime/blob/8b09702b88a2ed2258c5e5a5f63a5b539ac93f43/onnxruntime/core/session/inference_session.cc#L2050.